src/core/server/environment/environment_service.ts

```diff
@@ -77,6 +88,24 @@ export class EnvironmentService {
       logger: this.log,
     });
 
+    analytics.registerContextProvider({
+      name: 'kibana info',
+      context$: of({
+        kibana_uuid: this.uuid,
+        pid: process.pid,
+      }),
+      schema: {
+        kibana_uuid: {
+          type: 'keyword',
+          _meta: { description: 'Kibana instance UUID' },
+        },
+        pid: {
+          type: 'long',
+          _meta: { description: 'Process ID' },
+        },
```

Comment:
My initial idea was to identify pid changes as a way to identify 2 different processes (especially for the node-cluster initiative going on) of Kibana. I'm not entirely sure about how useful it be but I preferred to err on the side of having more info just in case.

If you think it's not worth it, I'm happy to remove it

Reply:
It's probably fine, I don't see any harm sending this